lms/djangoapps/support/static/support/jsx/success.jsx

```diff
@@ -0,0 +1,49 @@
+/* global gettext */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Success({homepageUrl, dashboardUrl, isLoggedIn}) {
+  let btnText,
+    btnUrl;
+  if (isLoggedIn) {
+    btnText = 'Go to my Dashboard';
+    btnUrl = {dashboardUrl}
+  } else {
+    btnText = 'Go to edX Home';
+    btnUrl = {homepageUrl}
+  }
+  return (<div className="contact-us-wrapper">
+    <div className="row">
+      <div className="col-sm-12">
+        <h2>{gettext('Contact Us')}</h2>
+      </div>
+    </div>
+
+    <div className="row">
+      <div className="col-sm-12">
+        <p>{gettext('Thank you for submitting a request! We will contact you within 24 hours.')}</p>
+      </div>
+    </div>
+
+    <div className="row">
+      <div className="col-sm-12">
+        <a
+          href={btnUrl}
+          className="btn btn-secondary help-button"
+        >
+          {gettext(btnText)}
```

Comment:
`gettext` here doesn't really do anything than mark that a variable called `bntText` needs translations. You should need to wrap it.